packaging/docker/custom/Dockerfile

```diff
@@ -42,8 +42,8 @@ RUN if [ ! -z "${GF_INSTALL_PLUGINS}" ]; then \
         IFS=$OLDIFS; \
         if expr match "$plugin" '.*\;.*'; then \
             pluginUrl=$(echo "$plugin" | cut -d';' -f 1); \
-            pluginWithoutUrl=$(echo "$plugin" | cut -d';' -f 2); \
-            grafana-cli --pluginUrl "${pluginUrl}" --pluginsDir "${GF_PATHS_PLUGINS}" plugins install ${pluginWithoutUrl}; \
+            pluginInstallFolder=$(echo "$plugin" | cut -d';' -f 2); \

```

Comment:
Why rename it, doesn't `pluginWithoutUrl` actually refer to a plugin? That does seem to be the case, since the `else` branch refers to it as `plugin`.

Reply:
Weirdly not no.

If I do 

```
./bin/darwin-amd64/grafana-cli --pluginUrl="https://storage.googleapis.com/plugins-community/alexanderzobnin-zabbix-app/release/4.1.4/alexanderzobnin-zabbix-app-4.1.4.zip" plugins install "Arve has a big beard"
```

We'll get

```
Plugin URL: (https://storage.googleapis.com/plugins-community/alexanderzobnin-zabbix-app/release/4.1.4/alexanderzobnin-zabbix-app-4.1.4.zip)
installing Arve has a big beard @ 
from: https://storage.googleapis.com/plugins-community/alexanderzobnin-zabbix-app/release/4.1.4/alexanderzobnin-zabbix-app-4.1.4.zip
into: /Users/wbrowne/dev/grafana/data/plugins

✔ Installed Arve has a big beard successfully 

Restart Grafana after installing plugins. Refer to Grafana documentation for instructions if necessary.
```
![image](https://user-images.githubusercontent.com/1173299/113010385-138cd880-9179-11eb-89f5-ed2e302e602e.png)